drivers/azure/azure.go

```diff
@@ -312,6 +322,7 @@ func (d *Driver) SetConfigFromFlags(fl drivers.DriverOptions) error {
 	d.FaultCount = fl.Int(flAzureFaultDomainCount)
 	d.UpdateCount = fl.Int(flAzureUpdateDomainCount)
 	d.DiskSize = fl.Int(flAzureDiskSize)
+	d.NSG = fl.String(flAzureNSG)
```

Comment:
why not do `d.resolveNSGReference()` here instead of create/remove?

Reply:
good idea, will add to the next commit